buffet/src/utils/createYupSchema.js

```diff
@@ -0,0 +1,82 @@
+import { isBoolean, isNaN } from 'lodash';
+import * as yup from 'yup';
+
+const createYupSchema = (type, validations, translatedErrors = {}) => {
+  let schema = yup.mixed();
+
+  if (['text', 'textarea', 'email', 'password'].includes(type)) {
+    schema = yup.string(translatedErrors.string);
+  }
+
+  if (type === 'email') {
+    schema = schema.email(translatedErrors.email);
+  }
+
+  if (type === 'number') {
```

Comment:
What about `bigInteger`, `integer`, `float`, `decimal`? See https://strapi.io/documentation/3.x.x/guides/models.html#define-the-attributes

Reply:
It concerns the input type (to define validation rules)